src/app.controller.spec.ts

```diff
@@ -6,7 +6,7 @@ import { RequestContext } from './shared/request-context/request-context.dto';
 
 describe('AppController', () => {
   let moduleRef: TestingModule;
-  const mockedLogger = { setContext: jest.fn(), log: jest.fn() };
+  const mockedLogger = { setContext: jest.fn(), logWithContext: jest.fn() };
```

Comment:
I think it's better to initialize these in beforeEach so that the mocks are clean for each test.

Reply:
Alright, let's change it across all in a separate PR. 🙏